packages/financial-templates-lib/src/logger/SlackTransport.js

```diff
@@ -75,13 +75,13 @@ function slackFormatter(info) {
         // For each key value pair within the object, spread the object out for formatting.
         for (const subKey in info[key]) {
           // If the length of the value is 66 then we know this is a transaction hash. Format accordingly.
-          if (info[key][subKey].length == 66) {
+          if (info[key][subKey]?.length == 66) {
```

Comment:
@chrismaree does this fix the slack log error?

Reply:
should work!